### ISSUE TITLE
Unable to save data sets 

### ISSUE THREAD

**Issue by ChrisBoehmCA** [zowe/vscode-extension-for-zowe#165]
Tested with three users on the newest version this morning. Data sets are not saving with no indication of what went wrong

I see this in zowe.log... the last line may be the clue. Maybe "path.relative" is not a reliable way to check whether you are in the temporary directory?

```
[2019/07/23 11:29:16.337] [DEBUG] [extension.js:102] File was saved -- determining whether the file is a USS file or Data set.
 Comparing (case insensitive) c:\Users\xxx\.vscode\extensions\zowe.vscode-extension-for-zowe-0.23.1\resources\temp\_D_\ca32\xxx.PUBLIC.CNTL(APARLIB).jcl against directory C:\Users\xxxx\.vscode\extensions\zowe.vscode-extension-for-zowe-0.23.1\resources\temp\_D_andC:\Users\xxxx\.vscode\extensions\zowe.vscode-extension-for-zowe-0.23.1\resources\temp\_U_
[2019/07/23 11:29:16.344] [DEBUG] [extension.js:107] File is a data set-- saving 
[2019/07/23 11:29:16.345] [DEBUG] [extension.js:1343] requested to save data set: c:\Users\xxx\.vscode\extensions\zowe.vscode-extension-for-zowe-0.23.1\resources\temp\_D_\ca32\xxxx.PUBLIC.CNTL(APARLIB).jcl
[2019/07/23 11:29:16.346] [DEBUG] [extension.js:1345] requested to save data set: c:\Users\xxxx\.vscode\extensions\zowe.vscode-extension-for-zowe-0.23.1\resources\temp\_D_\ca32\xxxxPUBLIC.CNTL(APARLIB).jcl
[2019/07/23 11:29:16.347] [DEBUG] [extension.js:1347] path.relative returned a non-blank directory.Assuming we are not in the DS_DIR directory: ..
```

**Comment by ChrisBoehmCA** [comment]
Jason tested with 0.23.0 as well which seems to have the same problem

**Comment by CoderCoco** [comment]
@zFernand0 says this was fixed in #166 but is still unpublished. Can a new release be published with this fix?
